extensions/argsparser.py

```diff
@@ -0,0 +1,39 @@
+import os
+import argparse
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        add_help=False,
+    )
+    parser.add_argument('objective', nargs='*', metavar='<objective>', help='''
+    main objective description. Doesn\'t need to be quoted.
+    if not specified, get objective from environment.
+    ''', default=[os.getenv("objective", "")])
+    parser.add_argument('-t', '--task', metavar='<initial task>', help='''
+    initial task description. must be quoted.
+    if not specified, get initial_task from environment.
+    ''', default=os.getenv("initial_task", os.getenv("FIRST_TASK", "")))
+    parser.add_argument('-4', '--gpt-4', dest='use_gpt4', action='store_true', help='''
```

Comment:
Curious, why this versus allowing model specification (text-davinci vs gpt-3.5)?

Reply:
For simplicity of usage. We can add separately model specification on .env for more advanced cases.

Reply:
Merged with the OPANAI_API_MODEL support in .env, so now both approaches work like everything else.